doc/preprocessing.rst

```diff
@@ -370,6 +370,15 @@ argument to `convert_mesh.py` script::
 
   script/convert_mesh.py -m screwdriver_full.msh screwdriver_full.vtk
 
+In order to extract the cells by the physical groups use the conversion script
+with ``--save_per_mat`` (``-pm``) argument::
+
+  script/convert_mesh.py -pm screwdriver_full.vtk screwdriver.vtk
+
+It produces `screwdriver_matid_1.vtk` and `screwdriver_matid_2.vtk` files
+containing only the cells of a given physical group and all vertices of the
+original mesh.
+
```

Comment:
It saves those meshes in addition to the output mesh - maybe mention that?

Reply:
Yes, I added the comment to this.